### DISCUSSION TITLE
HTPasswd - This is developer only feature and not recommended for use in Production?

### DISCUSSION THREAD

**Discussion by casaroto** [ibm-messaging/mq-container#498] [General]
Hello there,

Please could someone help me? 
On the Queue Manager Connection Authentication using a htpasswd file is written This is developer only feature and not recommended for use in Production. It still holds true?
Or there is some update of status? Is now a mature feature to use on IBM MQ 9.2.5?

**Comment by arthurbarr** [comment]
The code for using the HTPasswd file is all in this repository, and you are free to use it if you wish.  It is not supported by IBM, and has not been rigorously tested for production levels of security.
